mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,55 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  enlargeImage?: boolean;
+
+  /**
+   * Enlarge Image Styles
+   */
+  enlargeClassName?: string;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
   const className = clsx(styles.image, props.className);
+  const enlargeClassName = clsx(styles.image, props.enlargeClassName);
 
```

Comment:
Let's create one className
styles that can be not used you can describe as
 props.zoomable && styles.zoomable

Reply:
I can't understand this problem